packages/toolpad-app/src/components/AppEditor/PageEditor/QueryEditor.tsx

```diff
@@ -283,11 +284,36 @@ function QueryNodeEditorDialog<Q, P>({
     { retry: false },
   );
 
+  const untransformedQueryPreview = client.useQuery(
+    'execQuery',
+    input
+      ? [
+          appId,
+          {
+            ...input,
+            attributes: {
+              ...input.attributes,
+              transform: { value: '', type: 'const' },
```

Comment:
Shouldn't just overwriting `transformEnabled` be enough?

```suggestion
```

Reply:
Maybe we should abstract this in a `withTransformDisabled(input)` function?

Reply:
No, since then any change to the `transform` causes a refetch to get triggered